site-service/src/test/java/uk/ac/ox/ndph/mts/site_service/repository/FhirContextWrapperTests.java

```diff
@@ -0,0 +1,28 @@
+package uk.ac.ox.ndph.mts.site_service.repository;
+
+import ca.uhn.fhir.rest.server.exceptions.ResourceNotFoundException;
+import org.hl7.fhir.r4.model.Organization;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Test class for {FhirContextWrapper} note this should normally not be run because it depends on a particular
+ * FHIR API server being up
```

Comment:
So maybe we should just delete this?

Reply:
It's really handy for seeing what the API actually returns. Is there a place we can stash tools like this? I agree it should preferably not be in the test package.

Reply:
No, I removed application insights since it caused some errors with SLF4J. There's a better way to introduce it but will save that for a separate PR. I forgot to remove the azure-bom but did it now. Thanks.

Reply:
I've stashed it out of tree for now.